src/main/resources/db/changelog/changes/initial_schema.sql

```diff
@@ -51,7 +51,8 @@ CREATE TABLE IF NOT EXISTS retrieval_queue
     patron_name         TEXT,
     pickup_location     TEXT,
     request_status      TEXT,
-    request_note        TEXT
+    request_note        TEXT,
+    request_type        TEXT
```

Comment:
New field should be added via changelog-master.xml to database.

Reply:
changelog-master.xml has been updated.